### ISSUE TITLE
Strike3 not working with graphql

### ISSUE THREAD

**Issue by phoenixissue** [PAhelper/PhoenixAdult.bundle#1757]
**Describe the bug**
Strike3 websites not displaying any results. GET request gives error 403
If we use search?q= instead of graphql?query= it works but i find that the search gives a wide range of results

**Include**
1. [Scene URL](https://www.vixen.com/videos/i-have-a-confession-to-make)
2. File Name: Vixen - I have a confession to make.mp4
3. Log File
```
2023-03-29 21:06:52,876 (ffff9f234b08) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCnhuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM0CjgwNTdzMgppZHMzNQpWaXhlbiAtIEkgSGF2ZSBhIENvbmZlc3Npb24gdG8gTWFrZXM0Cm5hbWVuczQKeWVhcnIwCg__
2023-03-29 21:06:52,878 (ffff9f234b08) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2023-03-29 21:06:52,880 (ffff9f234b08) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCnhuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM0CjgwNTdzMgppZHMzNQpWaXhlbiAtIEkgSGF2ZSBhIENvbmZlc3Npb24gdG8gTWFrZXM0Cm5hbWVuczQKeWVhcnIwCg__
2023-03-29 21:06:52,881 (ffff9f234b08) :  INFO (agentkit:961) - Searching for matches for {'year': None, 'id': '8057', 'name': 'Vixen - I Have a Confession to Make'}
2023-03-29 21:06:52,882 (ffff9f234b08) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/8057/tree'
2023-03-29 21:06:52,901 (ffff9f234b08) :  INFO (__init__:68) - ***MEDIA TITLE [from media.name]*** Vixen - I Have a Confession to Make
2023-03-29 21:06:52,902 (ffff9f234b08) :  INFO (PAsearchSites:51) - mediaTitle w/ possible abbreviation: Vixen - I Have a Confession to Make
2023-03-29 21:06:52,930 (ffff9f234b08) :  INFO (PAsearchSites:59) - mediaTitle w/ possible abbrieviation fixed: Vixen - I Have a Confession to Make
2023-03-29 21:06:52,950 (ffff9f234b08) :  INFO (PAsearchSites:71) - ^^^^^^^ siteNum: 52
2023-03-29 21:06:52,951 (ffff9f234b08) :  INFO (PAsearchSites:72) - ^^^^^^^ Shortening Title
2023-03-29 21:06:52,960 (ffff9f234b08) :  INFO (PAsearchSites:96) - Search Title (before date processing): I Have a Confession to Make
2023-03-29 21:06:52,961 (ffff9f234b08) :  INFO (__init__:70) - {'siteName': 'vixen', 'searchDate': None, 'siteNum': 52, 'searchTitle': 'I Have a Confession to Make'}
2023-03-29 21:06:52,961 (ffff9f234b08) :  INFO (PAsearchData:23) - SearchData.title: I Have a Confession to Make
2023-03-29 21:06:52,962 (ffff9f234b08) :  INFO (__init__:107) - Provider: networkStrike3
2023-03-29 21:06:53,931 (ffff9f234b08) :  INFO (PAutils:183) - Requesting GET "http://www.vixen.com/graphql?query={searchVideos(input:{query:"I%20Have%20a%20Confession%20to%20Make",site:VIXEN,first:10}){edges{node{videoId,title,releaseDate,slug}}}}"
2023-03-29 21:06:54,027 (ffff9f234b08) :  INFO (PAutils:131) - FlareSolverr
2023-03-29 21:06:54,902 (ffff9f234b08) :  INFO (PAutils:183) - Requesting POST "http://localhost:8191/v1"
2023-03-29 21:06:54,908 (ffff9f234b08) :  INFO (PAutils:138) - CloudScraper
2023-03-29 21:06:55,032 (ffff9f234b08) :  INFO (PAutils:145) - ReqBin
2023-03-29 21:06:55,958 (ffff9f234b08) :  INFO (PAutils:183) - Requesting POST "https://api.reqbin.com/api/v1/requests"
2023-03-29 21:06:56,401 (ffff9f234b08) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2023-03-29 21:06:56,402 (ffff9f234b08) :  DEBUG (runtime:924) - Response: [200] str, 320 bytes
```

**Comment by diamondpete** [comment]
Please download the latest code. This was fixed with #1735.

http://www.vixen.com should be https://www.vixen.com with the latest code.

**Comment by phoenixissue** [comment]
> Please download the latest code. This was fixed with #1735.
> 
> http://www.vixen.com should be https://www.vixen.com with the latest code.

The latest code fixed it, thanks
